<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teeko</class>
 <widget class="QMainWindow" name="Teeko">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teeko</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget { background: url(':/grid'); }</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="Hole" name="hole20">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Hole" name="hole12">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="Hole" name="hole43">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>4</number>
      </property>
      <property name="col" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="Hole" name="hole40">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>4</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Hole" name="hole11">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="4">
     <widget class="Hole" name="hole04">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="Hole" name="hole00">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="4">
     <widget class="Hole" name="hole14">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="4">
     <widget class="Hole" name="hole34">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>3</number>
      </property>
      <property name="col" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="Hole" name="hole41">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>4</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="Hole" name="hole31">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>3</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="Hole" name="hole23">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="Hole" name="hole13">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="Hole" name="hole22">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="Hole" name="hole32">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>3</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Hole" name="hole10">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="4">
     <widget class="Hole" name="hole24">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Hole" name="hole02">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="Hole" name="hole30">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>3</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="Hole" name="hole33">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>3</number>
      </property>
      <property name="col" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="Hole" name="hole42">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>4</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Hole" name="hole01">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="Hole" name="hole03">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Hole" name="hole21">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="Hole" name="hole44">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>4</number>
      </property>
      <property name="col" stdset="0">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJogo">
    <property name="title">
     <string>Jogo</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuJogo"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="action9holes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9 Buracos</string>
   </property>
  </action>
  <action name="action13holes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>13 Buracos</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hole</class>
   <extends>QPushButton</extends>
   <header location="global">Hole.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
